docs/reference/release-notes/6.8.asciidoc

```diff
@@ -1,3 +1,75 @@
+[[release-notes-6.8.3]]
+== {es} version 6.8.3
+
+[[enhancement-6.8.3]]
+[float]
+=== Enhancements
+
+CCR::
+* Include leases in error message when operations no longer available {pull}45681[#45681]
+
+Infra/Core::
+* Add OCI annotations and adjust existing annotations {pull}45167[#45167] (issues: {issue}45162[#45162], {issue}45166[#45166])
+
+Infra/Settings::
+* Normalize environment paths {pull}45179[#45179] (issue: {issue}45176[#45176])
+
+
+
+[[bug-6.8.3]]
+[float]
+=== Bug fixes
+
+Aggregations::
+* Fix early termination of aggregators that run with breadth-first mode {pull}44963[#44963] (issue: {issue}44909[#44909])
+
+Allocation::
+* Avoid overshooting watermarks during relocation {pull}46079[#46079] (issue: {issue}45177[#45177])
+
+CCR::
+* Clean up ShardFollowTasks for deleted indices {pull}44702[#44702] (issue: {issue}34404[#34404])
+
+CRUD::
+* Allow _update on write alias {pull}45318[#45318] (issue: {issue}31520[#31520])
+
+Distributed::
+* Ensure AsyncTask#isScheduled remain false after close {pull}45687[#45687] (issue: {issue}45576[#45576])
+
+Engine::
+* Handle delete document level failures {pull}46100[#46100] (issue: {issue}46083[#46083])
+* Handle no-op document level failures {pull}46083[#46083]
+
+Features/Indices APIs::
+* Check shard limit after applying index templates {pull}44619[#44619] (issues: {issue}34021[#34021], {issue}44567[#44567], {issue}44619[#44619])
+* Prevent NullPointerException in TransportRolloverAction {pull}43353[#43353] (issue: {issue}43296[#43296])
+
+Features/Java High Level REST Client::
+* Add rest_total_hits_as_int in HLRC's search requests {pull}46076[#46076] (issue: {issue}43925[#43925])
+
+Infra/Core::
+* Add default CLI JVM options {pull}44545[#44545] (issues: {issue}219[#219], {issue}42021[#42021])
```

Comment:
It seems to me that #219 is unrelated

Reply:
It was a reference to an issue in elasticsearch-docker.